frontend/src/bundles/common/components/card-total/styles.module.scss

```diff
@@ -0,0 +1,64 @@
+:root {
+    --border-radius-1: 1px;
```

Comment:
can we use some default value?
4px
8px
12px
16px

Reply:
no border by default